### DISCUSSION TITLE
Add disable property to icons

### DISCUSSION THREAD

**Discussion by robert-hoffmann** [quasarframework/quasar#15778] [Ideas / Proposals] [answered]
It's not always practical to use buttons (size/color/padding), so having this feature on icons when we use them as buttons would be useful

**Comment by rstoenescu** [comment, marked as answer]
Hi,

When using QIcon, you're in charge of attaching the click handler. So you can decide if you actually execute it or not.
Doesn't quite makes sense to bloat QIcon with this functionality.

**Reply by robert-hoffmann** [reply]
I already do that on the click handler.

But unless i use a v-if with 2 different states (or cursor classes), i don't get the visual feedback like i get when a button is disabled

![image](https://user-images.githubusercontent.com/5472296/235638270-37dfba67-5169-4ea9-a222-a78236e597d5.png)

Sorry couldn't screenshot this one

![image](https://user-images.githubusercontent.com/5472296/235638965-fe4ae206-bf8a-4ea8-8026-38f6c6c371f2.png)


----
Update:

Ok, so now i'm switching on cursor classes, which does the job, and is fine even if i don't get the lighter-grey style out of the box
